routing/src/cpi.rs

```diff
@@ -1,483 +1,377 @@
 // SPDX-License-Identifier: Apache-2.0
 // Copyright Open Network Fabric Authors
 
-//! Control-plane interface (CPI)
+//! Main processing functions of the CPI
```

Comment:
Nit: I'd like to keep the expanded “Control-plane interface” somewhere, so that readers can understand what CPI means. This was a good place to have it. Maybe like this?:

```suggestion
//! Main processing functions of the CPI (Control-plane interface)
```